service-front/docker/web/Dockerfile

```diff
@@ -22,6 +22,8 @@ RUN apk upgrade
 USER nginx
 COPY --chown=nginx:nginx service-front/public /web
 COPY --chown=nginx:nginx service-front/docker/web/etc /etc
+COPY --chown=nginx:nginx service-front/docker/web/nginx.conf /etc/nginx.conf
```

Comment:
should this be copied to /etc/nginx/nginx.conf?

Reply:
Now fixed.